helm-charts/codegen/README.md

```diff
@@ -23,7 +23,7 @@ helm install codegen codegen --set global.HUGGINGFACEHUB_API_TOKEN=${HFTOKEN} --
 
 ### IMPORTANT NOTE
 
-1. To use model `meta-llama/CodeLlama-7b-hf`, you should first goto the [huggingface model card](https://huggingface.co/meta-llama/CodeLlama-7b-hf) to apply for the model access first. You need to make sure your huggingface token has at least read access to that model.
+1. To use model `Qwen/Qwen2.5-Coder-7B-Instruct`, you should first goto the [huggingface model card](https://huggingface.co/Qwen/Qwen2.5-Coder-7B-Instruct) to apply for the model access first. You need to make sure your huggingface token has at least read access to that model.
```

Comment:
This description may need to get modified.

Reply:
done